### PR TITLE
Add documentation for operator Prometheus metrics.

### PR DESCRIPTION
**Pull request by v-shenoy**
Signed-off-by: Vighnesh Shenoy <vshenoy@microsoft.com>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/issues/3663, https://github.com/kedacore/charts/pull/317

### CODE REVIEW

File: content/docs/2.9/operate/prometheus.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ weight = 200
 
 ## Prometheus Exporter Metrics
 
+### Operator 
+
+The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080` at `/metrics`. The following metrics are being gathered:
+
+- `keda_operator_trigger_totals` - Total number of triggers per trigger type.
```
Comment (tomkerkhove) [inline content/docs/2.9/operate/prometheus.md:13]: Can you add the other existing metrics please? @JorTurFer Can you elaborate on the ones available from operator framework?

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/2.9/operate/prometheus.md (hunk 1 above)
Comment (v-shenoy) [reply content/docs/2.9/operate/prometheus.md:13]: image
There are way too many of them to add here. I think this is the link that mentions them - https://book.kubebuilder.io/reference/metrics-reference.html

Maybe I can link to this in the docs?

Comment (v-shenoy) [reply content/docs/2.9/operate/prometheus.md:13]: But as you can see, in the screenshot there are a few metrics that are exposed the go_memstat, go_info, etc. But these are not in the Kubebuilder docs, I am not sure where these are being exposed from.

Comment (tomkerkhove) [reply content/docs/2.9/operate/prometheus.md:13]: If the docs are versioned and we can show the exact metrics that we offer that's fine for me

Comment (v-shenoy) [reply content/docs/2.9/operate/prometheus.md:13]: I don't think the docs are versioned. These are for Kubebuilder v3.

Comment (v-shenoy) [reply content/docs/2.9/operate/prometheus.md:13]: These docs are for Kubebuilder v3, the docs for v1, v2 are hosted at a separate URL.

Comment (tomkerkhove) [reply content/docs/2.9/operate/prometheus.md:13]: Perfect!

**Review by tomkerkhove — APPROVED** [review]
LGTM, waiting for feature to merge

**Pull request merged**